Wireframe/index.html

```diff
@@ -8,25 +8,37 @@
Y+pJ+tSZtZtYmi62RCGIUZJUqlin/EoiWTEr2zCT4rAzpbEx3r69ch4ilbflCJ8OPZgFG6LFdtan1QvdD4nOBdbUaWVJKdJp+s6OXNwEE5wid9BiOsQy5VxMsmJGrmFn7TqPz8rbFf2790bj7SP92/cr74zvVt4dGd+/SwN0Dh9mRA+TRII3k7GBFQzSKYYJSrBNiRQTSWE7pPNktpZVxIT0MShVOjcoc26yYvmGkdoVTCiDj/BiRbj9h/7of1bevjt6v/Li6O3K2yNBKx8c6zQag8bOqWSpxgwFxbHDGrPSSQq4H6mPmeepC84NMXVKdaZUjJUaFlQa0xl8/jCuCEfvXtx+C8+O3r97cXRbLxwdaeUD00+dKWYM1iUuk74uXaaOzhDNl7CfWDKloTMEDTTKnnaOTDG4IsmuG2WzSBWb5BsGyfXyAyyDz3u9/v3fj47+/mLl9h8rf7y4/e5IDy8084HJcwRiMMxMid1vYNL+Qhuxcnp2wtEEE+kmMQXXkfALRksuf0kOfArZ/0rU2cFkZ0FBo1ixfMOS7ZLRlF779EUmMKN6CIpjtCaTkczTwcyLUZ9wg+wUST8L900jYtyA0cg/RlRuUiwntl6a/GTrFBVK3CxDNpOpXjlqYVgIf7Qw6qSTu+m8xQaWbloZmpqqz7qDx9m57MjUVCgl3hVTeB5xMzmHuNIJGJE65c9OHhUblolD/ZKL4XNQBRKkwCoDa1+JIX41wKhmJrffy/x0D786xvN2rVUolFmQTJkiyy8UCzUE9Z3plOClcaoDQmGNtwBKiG9uOf+zH/z0IJv64e/wjMN9ET6XLHqDpDpCCeZXmAiCwamO+lgdm9M9gCRp4+Ow68bcyVdOm3e0xVK+fKiwuPmDTIgV746U32paEx9knw63BBL53DMer9PO2vKq76pJEx+3ztmT4oI8vI4dDFf4JOWxWbz9KdlYG38h/3yFpG18eXi3PWUycY7yqMInPd9Y0v/AOYEKnyzZnImM/OBghU92xtEkn8BqhU92zulVPvHMiSp8suRxJM7RuSt8ZLK2JLI6eHk+Kp9Tz/0mD+WhbD5y/Uhw8dAHT086n0TXYdk66JHFwHvGxpqvJKBMPrwnnNVFOEbnLn51mj6EA2l8+LFBeoZbZ7fZZ1LjSOQUyfJjMzabY7TUf9YrHyX4SL1yjzrsDq/UEekYHoZjo4QaP0qMgB8co3yS3/1we6fdMKGN2nQgh/MMEB/Q0dOaHSTFNnwFLUji4w6LbdeRPtqImfBhndhT3msnx3QEmocsuviWVcqnP6nVaccq76QQdLYAf7YhyjfbB8l8R8npHG7ldpSrRD68uyfAggn028WOhIlZOHXETpDUb/sg+Vp7Cxyi0zvkHUyqf9rRz4fFc3WjPSma7pkZ4/keMcXhLHVvcxesFdwekDugAygtNqmPw16vd9jr6EFJZrbhlkDA6yUrLxu1g7PvVrHD0zCMPlX7sUtX2u5g+TGJxZiTKOAgcfKqdCwQbg47nWQGm8nYbRX9s+R/ZtL8T0/p/A+frnxLgfFFfey0150cSw7ibt3TxOsixKaMOXoKnd7TW0If3D3q89d00eYv2iFnf0vAOwot75n2BsaaPSgvSpJ/ds7QxQM/5rDpbA763CktJ+Dqk2PJMIAPK9kB3zw26FaIfwaLFP8QBu7mFq/OYbfbbJK7gCd2h2M0MOhkc2Uk8cFSv6DzowFphCSBOAOj3rPeidO7YuNyTyqYiNWEBwmaBJh0ASX7aGDVg3NoS1Z8KGeFacfE6b3sBGyaW0btCmhSINkdOi81JG3Fal9/Jc7wOsuQDw4QNupoUgzJoZvu19SLPPjYynGaZmdSAdz58CoVhyADzh64JD6Z03t5iHWkdv7lx+MPupcvP96xvfz48hXwefXh4/GrD2k2dFl8sqb3slAmH93Pxy9fHv/88dUrQCN8fKn7cHx8fKcIfPiw9wrwOb7z6hggHQsfX935qBM+fvj44fjOnaLw6c95QBZD7Eyqjzm+cwxOCHzQHfpwx3YMv16m4rk0/8O3aHP8F1PuHyQYm7Y5VCcwgixhO3ZHz7C2eDV3Pt7w5fNh/3cv18UILJOdg8M6dUY0RnSMtqx6lG/Ik1Fsznx0nhxbnrvY1/94nQIo8dGL8z6CQaJYz2rLqEM+EiJo7N7BcE5rjCw+cuemHsOX/74Ffj3E8K+lv/KJ4V8E08dIJAFIBRTZbwkPtvQ4xAWYJDusv7xj1GzyWn+dvQ6vZobHPG5uPtuTvvzpnTX9s49lXg9xhAKHMOdfM2POynHX42ZsRhzH4XO+gEnWQ9gTHhxrGfZOw+q9ZWzV6c5vk0PiI308jPfoyF4zDZAxlo6twpVwJLw977zs6Z2NBUOvh17/699twIHzR08irpP1iG9tFvlm5yKz5utx13o0osF/F2z/hzhdLyXE0tWdjb5/E3aExXcraGRhl7bx+dVLnt7ZWANTH3r9rz/xwp/kI72ucZ9rzcxZXVGXz2+OrJt98fialbvUNqQI+JAbc1n6dc0Z+88Wz2iAZPCKvs7OitsHPZc8vZuCDGI6Rv4PYfO9Go6bi1M+aNYPfOJmsB/47V93FQ2Q0z5KNN0zM3i2/6yjh6btOsggba7bV1cDPT3D/eFLXp2y9SEm1oA32rj5PycwN+cb91lPfK7Z+JzfP+dD675o3BVfixfPgNxErFM37Eb8mGQr/fRgwBbmk5vrNu9Yc54+LjcxI1MdLOIW/vyTzl1+P+IifmT1uxDn8sNrF4r4XUX+/jff76DhpEfcJSBvNICrDmD6bqC4uT5dtHfQxW+ncvN0AxPT2YrD5IcEQOQRc8X+ejy/Ku6v8v0zECxIrKQ4mR92QPxQzE0fKfAp8U0U0nS219wy3OLJ2H9eDQwP4nJclJZC520/V1RRRRVVVJEG/T+ruoElSWMzCwAAAABJRU5ErkJggg==" alt="An example of a branch workflow on github" />
```

Comment:
I think it will be better if you can create svg image and don't use svg code directly in your html.